data-cube-curation/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+name: data-cube-curation
+description: RDF Data Cube curation service
+
+version: 0.1.0
+appVersion: latest
```

Comment:
Let's use a proper version number here. Otherwise it will get hard to track versions and do rollbacks etc.

Reply:
It seems like there are no stable tags/version number right now. Should I use the sha256 digest of the container for now? Or tag an arbitrary v0.0.1?

Reply:
If you can I'd tag the v0.0.1, yes. Otherwise let's go with the digest.

Reply:
Done